homeassistant/components/notify/group.py

```diff
@@ -0,0 +1,65 @@
+"""
+Group platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.group/
+"""
+import collections
+import logging
+import voluptuous as vol
+
+from homeassistant.const import (CONF_PLATFORM, CONF_NAME, ATTR_SERVICE)
+from homeassistant.components.notify import (DOMAIN, ATTR_MESSAGE, ATTR_DATA,
+                                             BaseNotificationService)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_SERVICES = "services"
+
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_PLATFORM): "group",
+    vol.Required(CONF_NAME): vol.Coerce(str),
+    vol.Required(CONF_SERVICES): vol.All(cv.ensure_list, [{
+        vol.Required(ATTR_SERVICE): vol.Any(cv.string, None),
```

Comment:
any reason why this can be `None`? That way we won't know what to call right ? 

Also, should it be `cv.slug` as services are slugified ?